web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -193,8 +194,11 @@ export class BarChartComponent implements AfterViewInit {
    *    <g class='axis-y'></g>
    *    <text id='bar-chart-title'></text>
    *    <text id='y-axis-title'></text>
-   *    <rect class='focusBars'></rect>
-   *    <rect class='transBars'></rect>
+   *    <rect class='new-bars'></rect>
+   *    <rect class='trans-bars'></rect>
+   *    // Implemented in local function changeFocus, so that the bars doesn't
+   *    // cover the number.
+   *    <g id='tag></g>
```

Comment:
We're using the 4-space indentation, I think it worth to fix the indentation for the comment here too.

Reply:
I don't quite understand. The comment here is commenting the line under it, where it needs another indentation?

Reply:
Ah sorry I misread the indentation here.